ansible/playbooks/host-setup.yml

```diff
@@ -40,3 +40,31 @@
   environment: "{{ deployment_environment_variables | default({}) }}"
   roles:
     - host_setup
+
+- hosts: nova_compute_nodes
+  become: true
+  gather_facts: true
+  environment: "{{ deployment_environment_variables | default({}) }}"
+  tasks:
+    - name: Install open-iscsi and multipath on nova compute nodes
+      when:
+        - enable_iscsi | default(false) | bool
+      block:
+        - name: Install Packages
+          ansible.builtin.package:
+            name:
+              - open-iscsi
+              - multipath-tools
+            state: latest # noqa package-latest
```

Comment:
typo

Reply:
I am not seeing it, can you point it out?

Reply:
So I think we should rather change the package state to 
`"{{ iscsi_package_state | default('present') }}"`
to prevent accidental upgrades and iscsi session issues for existing mounts